pages/index.tsx

```diff
@@ -1,28 +0,0 @@
-import { GetStaticPropsResult } from "next"
-import { Layout, LayoutProps } from "components/layout"
-import { getMenus } from "lib/get-menus"
-import { PageHeader } from "components/page-header"
-
-interface IndexPageProps extends LayoutProps {}
-
-export default function IndexPage({ menus }: IndexPageProps) {
```

Comment:
Since we are removing the index, the index is 404 by default now. I think we either need to make something else the index or make this work based on the Drupal configured index and document configuring that as an additional step.

Reply:
You can configure the [`DRUPAL_FRONT_PAGE`](https://next-drupal.org/docs/environment-variables#optional) env variables, it will be picked up automatically by `[[...slug]].tsx`.

To prevent the 404 by default, I think we can keep the `index.tsx` and add text to it on how to replace the home page.

Reply:
<img width="717" alt="Home_-_Acquia_CMS" src="https://user-images.githubusercontent.com/124599/159697185-15087e94-317a-49b6-bb25-4fd8d5c41efa.png">

Reply:
Makes sense! I had totally forgotten about that configuration option.